### PR TITLE
[DOCS] Update "How to create an Expectation Suite with the Onboarding Data Assistant"

### PR DESCRIPTION
**Pull request by NathanFarmer**
Changes proposed in this pull request:
- DOC-373
- Update to use existing workflow

### Definition of Done
- [X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [X] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md [hunk 1]
```diff
@@ -35,23 +29,29 @@ great_expectations suite new --profile
...
-And this is the configuration that you will use for your Batch Request:
+And this is the configuration that you will use for your `BatchRequest`:
 
-```python file=../../../../tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.py#L76-L81
+```python file=../../../../tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.py#L76-L80
 ```
 
+:::caution
+The Onboarding Data Assistant will run a high volume of queries against your `Datasource`. Data Assistant performance 
+  can vary significantly depending on the number of batches, count of records per batch, and network latency. It is
```
Comment (Rachel-Reverie) [inline docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md]:
```suggestion
  can vary significantly depending on the number of Batches, count of records per Batch, and network latency. It is 
```

File: docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md [hunk 2]
```diff
@@ -61,56 +61,72 @@ this code example:
...
 
-```python file=../../../../tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.py#L140-L148
+```python file=../../../../tests/integration/docusaurus/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.py#L163
 ```
 
-Once you have our `SimpleCheckpoint`'s configuration defined, you can instantiate a `SimpleCheckpoint` and run it.  You
- will check the `"success"` key of the `SimpleCheckpoint`'s results to verify that your Expectation Suite worked.
+![Plot Metrics](../../../images/data_assistant_plot_metrics.png)
+
+Hovering over a data point will provide more information about the batch and its calculated Metric value.
```
Comment (Rachel-Reverie) [inline docs/guides/expectations/data_assistants/how_to_create_an_expectation_suite_with_the_onboarding_data_assistant.md]:
```suggestion
Hovering over a data point will provide more information about the Batch and its calculated Metric value.
```

**Review by Rachel-Reverie — CHANGES_REQUESTED** [review]
Just a few minor instances where a technical term wasn't capitalized.

**Review by AFineDayFor — APPROVED** [review]

**Review by rdodev — APPROVED** [review]

**Comment by anthonyburdi** [comment]
Force merging before official announcement, @NathanFarmer verified these tests pass.

**Pull request merged**